### ISSUE TITLE
Not installing on Cygwin

### ISSUE THREAD

**Issue by evandrocoan** [asciidoc-py/asciidoc-py2#142]
When I tried to install this with cgywin, it failed with the error:
```
Professional@PROFESSIONAL-PC:$ make install
Fixing CONF_DIR in asciidoc.py
Fixing CONF_DIR in a2x.py
/usr/bin/install -c -d //usr/local/bin
/usr/bin/install: cannot create directory ‘//usr’: Read-only file system
make: *** [Makefile:101: progdir] Error 1
```

After seeing this, I edited the `Makefile` and remove all `$(DESTDIR)/` variables. And it worked correctly. This was happening because the `$(DESTDIR)` variable was not defined on the `Makefile`.

**Comment by elextr** [comment]
Please note that due to Python 2 end of life, this version of Asciidoc is no longer actively supported.

**Comment by MasterOdin** [comment]
Fixed in the supported asciidoc/asciidoc-py3@d0022f173e7f2d471d4f82b955837ddfd31cafd4 though.

**Comment by evandrocoan** [comment]
Thanks!
